cmd/btfs/autoupdate.go

```diff
@@ -84,25 +85,26 @@ func update() {
 	sleepTimeSeconds := 20
 	version := "0.0.0"
 	autoupdateFlg := true
-
-	// Get current config file.
-	currentConfig, err := getConfigure(currentConfigPath)
-	if err == nil {
-		autoupdateFlg = currentConfig.AutoupdateFlg
-		version = currentConfig.Version
-		sleepTimeSeconds = currentConfig.SleepTimeSeconds
-	}
-
-	if !autoupdateFlg {
-		fmt.Println("Automatic update is not turned on")
-		return
-	}
-
 	routePath := fmt.Sprint(runtime.GOOS, "/", runtime.GOARCH, "/")
 
 	for {
 		time.Sleep(time.Second * time.Duration(sleepTimeSeconds))
 
+		// Get current config file.
+		currentConfig, err := getConfigure(currentConfigPath)
+		if err == nil {
+			autoupdateFlg = currentConfig.AutoupdateFlg
+			version = currentConfig.Version
+			sleepTimeSeconds = currentConfig.SleepTimeSeconds
+		} else {
+			version = btfs_version.CurrentVersionNumber
+		}
+
+		if !autoupdateFlg {
+			fmt.Println("Automatic update is not turned on")
```

Comment:
In for loop, this message will print every period. Maybe we don't need this repeated message. Better let Tom or Zhimin decide.

Reply:
@taiyangc Hey Eric, would you mind taking a look at this? This is for the ticket https://tronnetwork.atlassian.net/browse/BTFSINFRA-297, which is auto-update for custom built binaries ticket. Lukasz says that the message won't be repeated in this logic. Thx

Reply:
It won't repeatedly print because the entire loop/wait logic is aborted on `return`.

Reply:
As it stands, the autoupdateFlg comes from the config file. The only way for the user to flip it (either turn on, or off), is to modify the config file manually. Depending on the setting in the config, the autoupdate is ON or OFF.